.gitignore

```diff
@@ -1,5 +1,5 @@
 # Project-specific files
```

Comment:
Can we also consolidate `data/*` and `test/test_data/*`, unless there's a good reason to have both of them